tools/qontract_cli.py

```diff
@@ -576,6 +577,36 @@ def quay_mirrors(ctx):
     print_output(ctx.obj['options'], mirrors, columns)
 
 
+@get.command()
+@click.argument('cluster')
+@click.argument('namespace')
+@click.argument('kind')
+@click.argument('name')
+@click.pass_context
+def root_owner(ctx, cluster, namespace, kind, name):
+    settings = queries.get_app_interface_settings()
+    clusters = [c for c in queries.get_clusters(minimal=True)
+                if c['name'] == cluster]
+    oc_map = OC_Map(clusters=clusters,
+                    integration='qontract-cli',
+                    thread_pool_size=1,
+                    settings=settings,
+                    init_api_resources=True)
+    oc = oc_map.get(cluster)
+    obj = oc.get(namespace, kind, name)
+    root_owner = oc.get_obj_root_owner(namespace, obj, allow_not_found=True,
+                                       allow_not_controller=True)
+
+    # TODO(mafriedm): fix this
+    # do not sort
```

Comment:
Can you clarify why you don't want this sorted?

Reply:
sorting is a bit buggy and will not work with just any object